Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+@Library('defra-library@v-10') _
+
+buildNodeJs npmAuditFailOnIssues: false
```

Comment:
Assume there's an unpatchable vulnerability we're happy with npm audit would otherwise block?